ocs_ci/helpers/dr_helpers.py

```diff
@@ -1628,17 +1633,22 @@ def disable_dr_from_app(secondary_cluster_name):
             cmd = f"oc patch placement {name} -n {namespace}  -p '{params}' --type=json"
             run_cmd(cmd)
 
-    # Delete all drpc
-    run_cmd("oc delete drpc --all -A")
-    sample = TimeoutSampler(
-        timeout=300,
-        sleep=5,
-        func=run_cmd_verify_cli_output,
-        cmd="oc get drpc -A",
-        expected_output_lst="No resources found",
-    )
-    if not sample.wait_for_func_status(result=False):
-        raise Exception("All drpcs are not deleted")
+    time.sleep(60)
+
+    if rdr:
+        disable_dr_rdr(discovered_apps=False)
+    else:
+        # Delete all drpc
+        run_cmd("oc delete drpc --all -A")
+        sample = TimeoutSampler(
+            timeout=300,
+            sleep=5,
+            func=run_cmd_verify_cli_output,
+            cmd="oc get drpc -A",
+            expected_output_lst="No resources found",
+        )
+        if not sample.wait_for_func_status(result=False):
+            raise Exception("All drpcs are not deleted")
```

Comment:
Better to get the remaining undeleted drpcs and print those in the exception

Reply:
Yes, I will add that in the enhancement PR